python/testing.rst

```diff
@@ -43,9 +45,8 @@ The important things to note in this example are:
 * When testing that an exception is raised always use `~unittest.TestCase.assertRaises` as a context manager, as shown in line 10 of the above example.
 * If a test method completes, the test passes; if it throws an uncaught exception the test has failed.
 
-We write test files to allow them to be run by `pytest`_ or to allow them to be run directly from the command line using :command:`python`.
-Whilst `pytest`_ provides more flexibility and enhanced reporting when running tests (such as specifying that only certain tests run), it is sometimes expedient to run them using :command:`python` since there is a faster start time and all the output is visible.
-Additionally, although `pytest`_ allows `unittest.TestCase.subTest` to be used they are treated as normal tests and will fail immediately whereas running with :command:`python` the sub tests will all be executed before triggering stopping the test.
+We write test files to allow them to be run by `pytest`_ rather than simply :command:`python`, as the former provides more flexibility and enhanced reporting when running tests (such as specifying that only certain tests run).
+Note that while `pytest`_ allows `unittest.TestCase.subTest` to be used they are treated as normal tests and will fail immediately (whereas running with :command:`python` the sub tests will all be executed before triggering stopping the test).
```

Comment:
I don't think that's true now that we use `pytest-subtest`. The subtest code does cause some problems in that there is (or used to be) a bug where standard out is not captured from subtests.